ADALiOS/ADALiOS/ADAuthenticationContext+Internal.m

```diff
@@ -160,28 +158,4 @@ - (ADAuthenticationResult*)updateResult:(ADAuthenticationResult*)result
     return result;
 }
 
-//Makes sure that the correlation id contains a valid UUID.
```

Comment:
I found that the currently version of code has abandoned this function and use another function ensureRequest() to do the same thing. It is not used at all.

If we still need it, I guess we can add it back to ADAuthenticationContext.m because correlationId has been made as a property of ADAuthenticationContext.